klayout/drc/rule_decks/otp_mk.drc

```diff
@@ -0,0 +1,153 @@
+
+  # Rule O.SB.2: Min. salicide Block Space. is 0.28µm
+  logger.info('Executing rule O.SB.2')
+  osb2_l1 = sab_otp.space(0.28.um, euclidian)
+  osb2_l1.output('O.SB.2', 'O.SB.2 : Min. salicide Block Space. : 0.28µm')
+  osb2_l1.forget
+
+  # Rule O.SB.3: Min. space from salicide block to unrelated COMP. is 0.09µm
+  logger.info('Executing rule O.SB.3')
+  osb3_l1 = sab.outside(comp).and(otp_mk).separation(comp.outside(sab), 0.09.um, euclidian)
+  osb3_l1.output('O.SB.3', 'O.SB.3 : Min. space from salicide block to unrelated COMP. : 0.09µm')
+  osb3_l1.forget
+
+  # Rule O.SB.4: Min. space from salicide block to contact.
+  logger.info('Executing rule O.SB.4')
+  osb4_l1 = sab_otp.separation(contact, 0.03.um, euclidian).polygons(0.001.um).or(sab_otp.and(contact))
+  osb4_l1.output('O.SB.4', 'O.SB.4 : Min. space from salicide block to contact.')
+  osb4_l1.forget
+
+  # Rule O.SB.5a is not a DRC check
+  ## Please refer to https://gf180mcu-pdk.readthedocs.io/en/latest/physical_verification/design_manual/drm_10_10.html
+
+  # Rule O.SB.5b_LV: Min. space from salicide block to unrelated Poly2 on COMP. is 0.1µm
+  logger.info('Executing rule O.SB.5b_LV')
+  sab_otp_lv = sab_otp.not_interacting(v5_xtor).not_interacting(dualgate)
+  osb5b_l1 = sab_otp_lv.outside(tgate).separation(tgate.outside(sab), 0.1.um, euclidian)
+  osb5b_l1.output('O.SB.5b_LV', 'O.SB.5b_LV : Min. space from salicide block to unrelated Poly2 on COMP. : 0.1µm')
+  osb5b_l1.forget
+  sab_otp_lv.forget
+
+  # Rule O.SB.5b_MV is not a DRC check
+  ## Please refer to https://gf180mcu-pdk.readthedocs.io/en/latest/physical_verification/design_manual/drm_10_10.html
+
+  # Rule O.SB.9: Min. salicide block extension beyond unsalicided Poly2. is 0.1µm
+  logger.info('Executing rule O.SB.9')
+  osb9_l1 = poly_otp.enclosed(sab_otp, 0.1.um, euclidian)
+  osb9_l1.output('O.SB.9', 'O.SB.9 : Min. salicide block extension beyond unsalicided Poly2. : 0.1µm')
+  osb9_l1.forget
+
+  # Rule O.SB.11: Min. salicide block overlap with COMP. is 0.04µm
+  logger.info('Executing rule O.SB.11')
+  osb11_l1 = sab_otp.overlap(comp, 0.04.um, euclidian)
+  osb11_l1.output('O.SB.11', 'O.SB.11 : Min. salicide block overlap with COMP. : 0.04µm')
+  osb11_l1.forget
+
+  # Rule O.SB.12 is not a DRC check
+  ## Please refer to https://gf180mcu-pdk.readthedocs.io/en/latest/physical_verification/design_manual/drm_10_10.html
+
+  # Rule O.SB.13_LV: Min. area of silicide block (um2). is 1.488µm²
+  logger.info('Executing rule O.SB.13_LV')
+  osb13_l1 = sab_otp.with_area(nil, 1.488.um).not_interacting(v5_xtor).not_interacting(dualgate)
+  osb13_l1.output('O.SB.13_LV', 'O.SB.13_LV : Min. area of silicide block (um2). : 1.488µm²')
+  osb13_l1.forget
+
+  # Rule O.SB.13_MV: Min. area of silicide block (um2). is 2µm²
+  logger.info('Executing rule O.SB.13_MV')
+  osb13_l1 = sab_otp.and(v5_xtor).with_area(nil, 2.um)
```

Comment:
shouldn't that be `sab_otp.and(v5_xtor).with_area(2.um, nil)` ?

Reply:
Same

Reply:
#54